ui/partials/perf/alertsctrl.html

```diff
@@ -170,6 +170,14 @@
         </td>
       </tr>
     </table>
+    <div class="panel-body" ng-show="alertSummary.downstreamSummaryIds.length">
+      <p class="text-muted">
+        Downstream alert summaries:
+        <span class="text-muted" ng-repeat="summaryId in alertSummary.downstreamSummaryIds">
+          <a href="perf.html#/alerts?id={{summaryId}}">#{{summaryId}}</a>
```

Comment:
I think this would look a little better with a comma seperating the elements. Could you use the technique described here to do that?

http://stackoverflow.com/a/11541161